app/infrastructure/database/repositories.py

```
@@ -2,134 +1,91 @@
-        pass
-
-    @abstractmethod
-    async def update_last_used(self, api_key_id: str) -> None:
-        pass
-
-
-class TortoiseAPIKeyRepository(APIKeyRepositoryInterface):
-    async def find_active_by_key(self, api_key: str) -> Optional[APIKEY]:
+class APIKeyRepositoryHelper:
+    
+    def __init__(self, repo=TortoiseApiKeyStore()):
```

Ruff (0.12.2)

Do not perform function call `TortoiseApiKeyStore` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)